test/helpers/web3/preloadWeb3.js

```diff
@@ -0,0 +1,24 @@
+const Ganache = require(process.env.TEST_BUILD
+  ? "../build/ganache.core." + process.env.TEST_BUILD + ".js"
+  : "../../../index.js");
+const Web3 = require("web3");
+
+const preloadWeb3 = (options = {}) => {
```

Comment:
As I said elsewhere, I don't understand the functional difference between this module and `getWeb3` - seems like a pretty direct duplication of code

Reply:
preloadWeb3 sets up a provider within a mocha `before` hook